src/BenchmarkDotNet/Templates/CsProj.txt

```diff
@@ -11,9 +11,12 @@
     <OutputType>Exe</OutputType>
     <OutputPath>bin\$CONFIGURATIONNAME$</OutputPath>
     <TreatWarningsAsErrors>False</TreatWarningsAsErrors>
-    <!-- nice to have for EtwProfiler, but not mandatory. On x86 and ARM disabled due to bug in Roslyn. -->
-    <DebugType Condition=" '$(PlatformTarget)' != 'x86' And '$(PlatformTarget)' != 'ARM' ">pdbonly</DebugType>
-    <DebugSymbols Condition=" '$(PlatformTarget)' != 'x86' And '$(PlatformTarget)' != 'ARM' ">true</DebugSymbols>
+    <!-- nice to have for EtwProfiler, but not mandatory.
```

Comment:
I'd recommend removing DebugType and DebugSymbols properties altogether and use the defaults (i.e. Portable PDBs)